hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/transfer/AdjustHbarChangesStepTest.java

```diff
@@ -114,6 +116,64 @@ void doesHbarBalanceChangesWithAllowances() {
         assertThat(senderAccountAfter.cryptoAllowances()).isEmpty();
     }
 
+    @Test
+    void doesHbarBalanceChangesWithLeftoverAllowances() {
```

Comment:
'does' with 'changes' conjugation

```suggestion
    void hbarBalanceChangesWithLeftoverAllowances() {
```

Reply:
I agree but the existing tests are all phrased as 'doesHbarSomething' so I kept the pattern.